aws_lambdas/twilio_webhook.py

```diff
@@ -0,0 +1,59 @@
+import json
+import logging
+import os
+from typing import Any, Dict
+from urllib.parse import unquote_plus
+
+import boto3
+from twilio.request_validator import RequestValidator
+
+from stopcovid.dialog.command_stream.publish import CommandPublisher
+from stopcovid.utils.logging import configure_logging
+
+configure_logging()
+
+
+def handler(event, context):
+    kinesis = boto3.client("kinesis")
+    stage = os.environ["STAGE"]
+
+    form = extract_form(event)
+    if not is_signature_valid(event, form, stage):
+        logging.warning("signature validation failed")
+        return {"statusCode": 403}
+    if "MessageStatus" in form:
+        logging.info(f"Outbound message to {form['To']}: Recording STATUS_UPDATE in message log")
+        kinesis.put_record(
+            Data=json.dumps({"type": "STATUS_UPDATE", "payload": form}),
+            PartitionKey=form["To"],
```

Comment:
For my own sanity:

- Message status events are published after twilio receives a callback after sending an outbound message
- Outbound message means that our system is sending a message TO a user
- We should be using the "To" phone number
- This is correct